specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/preview/2015-06-15/Network.json

```diff
@@ -119,14 +215,29 @@
           "description": "The name of the operation being performed on this particular object. This name should match the name that appears in RBAC or the event service.",
           "type": "string",
           "readOnly": true
-        },
-        "display": {
```

Comment:
Is this supposed to be a non-existing property in older spec, or is it a breaking change (no longer backward compatible)?

Reply:
Good catch! This was removed by mistake. I have reverted this removal.